ee/ucp/deploy-application-package.md

```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
```

Comment:
Change to read, "Click to indidate the **Application File Mode**, either **Compose File** or **App Package**.